app/opinions/getNewestOpinions.js

```diff
@@ -0,0 +1,42 @@
+import express from "express";
+import bodyParser from "body-parser";
+import mongoose from "mongoose";
+
+const app = express();
+
+app.use(
+  bodyParser.urlencoded({
+    extended: true,
+  }),
+);
+app.use(express.static("public"));
+
+mongoose.connect("mongodb://localhost:27017/Opinions-db", {
+  useNewUrlParser: true,
+});
+
+const OpinionSchema = new mongoose.Schema({
+  rating: Number,
+  description: String,
+  name: String,
+  city: String,
+});
+
+const Opinions = mongoose.model("Opinions", OpinionSchema);
```

Comment:
Would be nice to put every model into separate file, let's say in `models/` directory.